### PR TITLE
Audit and update dependencies

### PR DESCRIPTION
**Pull request by rolandszoke**

### CODE REVIEW

File: __tests__/Clock.test.jsx [hunk 1]
```diff
@@ -12,9 +12,8 @@ describe('Clock App', () => {
   );
 
   const clearIntervalSpy = sinon.spy(global, 'clearInterval');
-
+  afterAll(cleanup);
```
Comment (solkimicreb) [inline __tests__/Clock.test.jsx]: Out of curiosity? Why was this `afterAll` split required?

Comment (rolandszoke) [reply __tests__/Clock.test.jsx]: I'm not sure, but `@testing-library/dom` changed, and If you do not add the `cleanup` directly to the `afterAll` or `afterEach`, but run it in a function (it does not matter if it is an arrow function or not) the `cleanup` will lose the scope of the `document` so it is unable to find the `body` and clean it up, and the function breaks.

Comment (rolandszoke) [reply __tests__/Clock.test.jsx]: Ideally, we could have our tests isolated so we would not need to use `cleanup` at all. Maybe I should start working on it.

**Pull request merged**